README.md

```diff
@@ -35,6 +35,7 @@ Customisable smart app banner for iOS and Android.
 <meta name="smartbanner:button-url-apple" content="https://ios/application-url">
 <meta name="smartbanner:button-url-google" content="https://android/application-url">
 <meta name="smartbanner:enabled-platforms" content="android,ios">
+<meta name="smartbanner:days-to-hide" content="15">
```

Comment:
I like the simplicity of the wording, but it defeats granularity that some may need. 

I'd suggest `smartbanner:hide-ttl` instead.